indico/modules/designer/templates/template.html

```diff
@@ -1,268 +1,315 @@
+                        {%- trans %}Insert{% endtrans -%}
+                    </button>
+                </div>
+            </div>
+        </div>
+        <table class="designer-table" border="0" cellpadding="0" cellspacing="0">
+            <tbody>
+                <tr>
+                    <td align="left" valign="bottom" height="16px">
+                        <div id="horizontal-ruler" class="ruler">
                         </div>
-                        <a id="remove-button"
-                           class="right i-button icon-remove icon-only"
-                           title="{% trans %}Remove Element{% endtrans %}"></a>
-                    </div>
-                </td>
-
-                <td></td>
-
-                <td align="left" valign="bottom" height="22px">
-                    <div id="horizontal-ruler" class="ruler">
-                    </div>
-                </td>
-            </tr>
-            <tr>
-                <td valign="top" align="right" width="22px">
-                    <div id="vertical-ruler" class="ruler">
-                    </div>
-                </td>
-
-                <td align="left" valign="top">
-                    <div id="template-div">
-                        <img id="background">
-                    </div>
-                </td>
-            </tr>
-        </tbody>
-    </table>
+                    </td>
+                </tr>
+                <tr>
+                    <td valign="top" align="right" width="16px">
+                        <div id="vertical-ruler" class="ruler">
+                        </div>
+                    </td>
+                    <td align="left" valign="top">
+                        <div class="template-container">
+                            <div class="template-content">
+                                <div class="template-side front active" data-side="front">
+                                    <img class="background-image">
+                                </div>
+                                <div class="template-side back" data-side="back">
+                                    <div class="back-side-placeholder">
+                                        <div class="placeholder-content">
+                                            <div>{% trans %}The back side is empty.{% endtrans %}</div>
+                                            <div>
+                                                {% set link_start %}
+                                                    <a href="#" class="js-backside-list-dialog"
+                                                       data-href="{{ url_for('.backside_template_list', template) }}"
+                                                       data-title="{% trans %}Select a template for the back side{% endtrans %}">
+                                                {% endset %}
+                                                {%- trans link_start=link_start|safe, link_end='</a>'|safe -%}
```

Comment:
While technically the same I would use `{% set link_end %}</a>{% endset %}` above. Otherwise editors like to screw up closing tag detection and highlighting since they see the link start tag but not take the quoted end tag into account.

Editor quirks aside, this would also be more consistent since the start/end tag are set in the same way